### PR TITLE
LeaderBoard: Setup Project

### PR DESCRIPTION
**Pull request by manzitresor**
### LeaderBoard: Setup Project
The leaderboard website displays scores submitted by different players. It also allows you to submit your score.

### Implemented Features

- [ ] Created an npm project with webpack. 
- [ ] Written plain HTML markup with minimum styling
- [ ] Use ES6 modules, with import and export.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,47 @@
+{
+  "name": "todolist_review",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "build": "webpack",
+    "dev": "webpack serve",
+    "test": "jest"
+  },
```
Comment (rloterh) [inline package.json]: - [ ] Let's ensure that the app can run correctly by adding the appropriate parameters to the _script_ section of the _package.json_, like the below:

```javascript
  "scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack serve --mode development"
  },
```

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Manzi,  

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 
🏁  🌴 🚦 🚗 🏫  🏡   _🏃 🚜  ⛪ 

## Highlights 👍  
- Linters setup correctly!
- ReadMe looks good!

## Required Changes ♻️
_Kindly check the inline comments under the review._

## Optional suggestions 🧰 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

**You can also consider:**
Kindly include instructions in your ReadMe on how to run your project, under the _Getting Started_ section.
Implementing the above change(s) will make your ReadMe/ repository stand out even better! 😉

Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by manzitresor** [comment]
Hey @rloterh thank you so much for review feedback 🤝 
I will take required changes into consideration and workon it.

**Review by emmiiorji — APPROVED** [review]
Hi @manzitresor, 

# Status: Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

<div align='center'>

</div>

## Good Highlights✔️

- Requested change has been implemented

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by manzitresor** [comment]
Hey @emmiiorji Thank you so much for approving my Project 🤝 
it motivated me, cheers 🍾

**Pull request merged**